e2e/tests/canvas.spec.ts

```
@@ -0,0 +1,152 @@
+            const graph = await browser.createNewPage(GraphPage, urls.graphUrl);
+            await browser.setPageToFullScreen();
+            const graphName = getRandomString('canvas');
+            await graph.addGraph(graphName);
+            await graph.insertQuery(BATCH_CREATE_PERSONS);
+            await graph.clickRunQuery();
+            const searchQuery = `Person ${node}`;
+            await graph.searchForElementInCanvas(searchQuery);
+            await graph.hoverAtCanvasCenter();
+            expect(await graph.getNodeCanvasToolTip()).toBe(searchQuery);
+            await apicalls.removeGraph(graphName);
+        });
```

ESLint

[error] Function declared in a loop contains unsafe references to variable(s) 'browser', 'browser', 'apicalls'.

(@typescript-eslint/no-loop-func)